runtime/help/plugins.md

```diff
@@ -357,8 +357,9 @@ This file will contain the metadata for your plugin. Here is an example:
 }]
 ```
 
-Then open a pull request at github.com/micro-editor/plugin-channel adding a link
-to the raw `repo.json` that is in your plugin repository. To make updating the
-plugin work, the first line of your plugins lua code should contain the version
-of the plugin. (Like this: `VERSION = "1.0.0"`) Please make sure to use
-[semver](http://semver.org/) for versioning.
+Then open a pull request at
+[micro-editor/plugin-channel](https://github.com/micro-editor/plugin-channel),
```

Comment:
looks good to merge.

thanks. have a nice day. keep up the great work. 👏

[ret394](https://github.com/ret394)